x11-packages/tuxpaint/build.sh

```diff
@@ -33,8 +35,12 @@ termux_step_pre_configure() {
 	# this is a workaround for build-all.sh issue
 	TERMUX_PKG_DEPENDS+=", tuxpaint-data"
 
-	local _PREFIX_FOR_BUILD=$TERMUX_PKG_HOSTBUILD_DIR/prefix
-	export PATH=$_PREFIX_FOR_BUILD/bin:$PATH
+	local _PREFIX_FOR_BUILD="$TERMUX_PKG_HOSTBUILD_DIR/prefix"
+	export PATH="$_PREFIX_FOR_BUILD/bin:$PATH"
+	export XDG_DATA_HOME="$TERMUX_PREFIX/share" XDG_DATA_DIRS="$TERMUX_PREFIX" XDG_CURRENT_DESKTOP="X-Generic"
+	
+	# Disabling gtk-update-icon-cache
+	ln -s /usr/bin/true "$_PREFIX_FOR_BUILD/bin/gtk-update-icon-cache" ||:
```

Comment:
Don't think I've seen that one before.
Why exactly are we disabling `gtk-update-icon-cache` here?

Reply:
I am not really sure. It is disabled everywhere during cross-build.
[1](https://github.com/termux/termux-packages/blob/152a3a06c11882b054823172b5c934d851c38bfb/x11-packages/azpainter/install.sh.in.patch) [2](https://github.com/termux/termux-packages/blob/152a3a06c11882b054823172b5c934d851c38bfb/x11-packages/xfdesktop/dont-update-icon-cache.patch) [3](https://github.com/termux/termux-packages/blob/152a3a06c11882b054823172b5c934d851c38bfb/x11-packages/gtk3/0002-dont-update-icon-cache.patch) [4](https://github.com/termux/termux-packages/blob/152a3a06c11882b054823172b5c934d851c38bfb/x11-packages/xfce4-panel/dont-update-icon-cache.patch) [5](https://github.com/termux/termux-packages/blob/152a3a06c11882b054823172b5c934d851c38bfb/x11-packages/xfce4-session/dont-update-icon-cache.patch)

Reply:
As you can see something here breaks build so I will investigate it more.

Reply:
Guessing it just doesn't apply to Termux.
Oh well.